USAGE.md

```diff
@@ -327,6 +327,30 @@ Potential spelling errors will be printed in the continuous integration log alon
 Words in `build/assets/custom-dictionary.txt` are ignored during spellchecking.
 Spellchecking is currently only supported for English language manuscripts.
 
+## AI-assisted authoring
+
+The workflow [`ai-revision`](.github/workflows/ai-revision.yaml) is available to assist authors in writing their manuscripts.
+It uses large language models to revise the manuscript text, fixing spelling and grammar errors, and improving the sentence structure and the writing style with section-specific prompts.
+It is manually triggered by the user (it never runs automatically), and it generates a pull request with suggested revisions.
+Then the user can review these changes and merge the pull request if they are acceptable.
+More information about this tool is available in [this manuscript](https://greenelab.github.io/manubot-gpt-manuscript/).
+
+You must provide a secret with name `OPENAI_API_KEY` containing your OpenAI API token.
+The easiest way to set this is to go to your organization's settings page and, within "Secrets and variables," select "Actions."
+Next, create a secret with the name `OPENAI_API_KEY` and the value of the API token.
+By default, the tool uses the model `text-davinci-003`.
+Make sure to check the [pricing](https://openai.com/api/pricing/) of the OpenAI API.
+With $0.02 per 1000 tokens using the most powerful AI models, the cost for a revision of a standard manuscript (around 35 paragraphs) should be around $0.50.
+The workflow allows specifying the branch and file names (in the `content/` directory) to revise, the language model to use, and the output branch name.
+Internally, the workflow uses the tool [Manubot AI Editor](https://github.com/greenelab/manubot-ai-editor) to revise the manuscript.
+For more advanced users, the behavior of the Manubot AI Editor or the parameters used for the language model can be changed using environment variables.
+These variables can be changed in the workflow file (`ai-revision.yaml`).
+
+It is important to note that using language models in scientific writing is a matter of debate among researchers and journal editors.
+Therefore, it's advisable to follow the guidelines that journals and the research community propose.
+For example, the *Nature* journal has published [rules about using language models in scholarly writing](https://www.nature.com/articles/d41586-023-00191-1), such as not listing the tools as authors and documenting how they were used.
+Since a Manubot-based manuscript uses GitHub, one approach consists in linking the AI-generated pull request, which will transparently show the changes suggested by the AI tool.
```

Comment:
```suggestion
Since a Manubot-based manuscript uses GitHub, one approach consists of linking the AI-generated pull request, which will transparently show the changes suggested by the AI tool.
```

I like your phrasing in this section. I don't want to add it to this section, but note that the new policy at Science prohibits using AI-generated text https://www.science.org/doi/10.1126/science.adg7879. It's not clear that AI-revised text would fall under that policy, and we shouldn't try to catalog all journals' policies here.

Reply:
We could add another sentence saying that authors should check the policy of the target journal before using this tool.

Reply:
I'd say
> it's advisable to follow the guidelines that journals and the research community propose.

covers that well enough already